app/controllers/cities_controller.rb

```diff
@@ -1,14 +1,34 @@
 class CitiesController < ApplicationController
 
+  # TOP 10 FR by population
+  MAIN_CITIES = %w(
+    Paris
+    Marseille
+    Bruxelles
+    Lyon
+    Toulouse
+    Nice
+    Nantes
+    Strasbourg
+    Montpellier
+    Bordeaux
+  )
+
   def autocomplete
     @results = City.search(
       params[:term],
       limit: 5,
-      boost_where: {country_code: {value: 'FR', factor: 5}},
+      boost_where: { country_code: { value: 'FR', factor: 5 } },
```

Comment:
Ideally, we would boost the city according to its population (and we would not need a hardcoded MAIN_CITIES). Population was in geoname DB but I'm not sure it's imported in our project.

Reply:
Yes, but we haven't yet the population in our DB.
Another plus would be to propose main cities around the IP by default before user tipped 2 characters.
Linked with the " Dépôt d'annonce: suggestion d'étapes " https://github.com/covoiturage-libre/covoiturage-libre/issues/212

Until done we can start with a hard coded list.

Reply:
Not optimal but you're right until then (would be cool to have a nice city DB with extra properties like population for interesting search / suggestions).
Good point of this hard-coded implementation is that it's fast, and it's certainly a must for the main input.